eppy/bunch_subclass.py

```diff
@@ -394,6 +400,18 @@ def __dir__(self):
         return super(EpBunch, self).__dir__() + fnames + func_names
 
 
+def _parse_idd_type(epbunch, name):
+    """parse the fieldvalue type into a python type. eg.: 'real' returns
+    'float'"""
+    _type = next(iter(epbunch.getfieldidd(name)['type']), None)
+    if _type == 'real':
```

Comment:
If you search an IDD for those last two, it might give you ideas on what we need do with them.

Also, watch out for "AUTOCALCULATE" value which can appear in fields with `real` (and maybe `integer`) type. Probably best to add it as a test case.

Reply:
This is handled in 7d3ccf4

Reply:
This isn't resolved as it doesn't yet handle "AUTOCALCULATE".

Reply:
Not sure I understand why returning the string "autocalculate" is not ok.

Reply:
It's used in fields which are marked as `real`, so will end up calling `float("AUTOCALCULATE")` which will raise a ValueError.